concatenate_flows/concatenate_pdms.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import csv
+import time
+import re
+from core_data_modules.traced_data import TracedData, Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO
+
+
+def key_normalisation(td_pdm, string_to_replace, replacement_string):
+    """
+    Used to normlise the keys from different PDMs. Acts to the TracedData
+    Object itself(td_pdm)
+
+    :param td_pdm: Data to normalise the keys of
+    :type td_pdm:TracedData object
+    :param string_to_replace: String to search for and replace
+    :type string_to_replace: str
+    :param replacement_string: String create new key from 
+    :type replacement_string: str
+    """
+    md =  Metadata(user, Metadata.get_call_location(), time.time())
```

Comment:
You need to initialise your Metadata at the same time as you append. Otherwise the timestamps will be incorrect and the source field in Metadata will be incorrect